public/utils/constants.ts

```diff
@@ -10,3 +11,8 @@ export enum TAB_ID {
   HISTORY = 'history',
   TRACE = 'trace',
 }
+
+export const DEFAULT_SIDECAR_DOCKED_MODE = SIDECAR_DOCKED_MODE.RIGHT;
+export const DEFAULT_SIDECAR_LEFT_OR_RIGHT_SIZE = 460;
+// this is a default padding top size for sidecar when switching to takeover
+export const DEFAULT_SIDECAR_TAKEOVER_PADDING_TOP_SIZE = 460;
```

Comment:
As I remember, this value should be 136 before. Is this value the same as `DEFAULT_SIDECAR_LEFT_OR_RIGHT_SIZE`?